app/server/openshift/src/main/java/io/syndesis/server/openshift/OpenShiftConfigurationProperties.java

```diff
@@ -27,13 +30,11 @@
 @Validated
 public class OpenShiftConfigurationProperties {
 
-    public static final String SERVICE_CA_CERT_FILE = "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt";
+    public String serviceCaCertFile = "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt";
 
     private boolean enabled;
 
-    private String openShiftHost = "https://openshift.default.svc";
-
-    private final OpenShiftConfig openShiftClientConfig = new OpenShiftConfigBuilder().withMasterUrl(openShiftHost).withCaCertFile(SERVICE_CA_CERT_FILE).build();
+    private String openShiftHost = "https://kubernetes.default.svc";
```

Comment:
Does this work on 3.11? (Thinking about back porting to 1.6.x)

Reply:
yes.  it works.